### PR TITLE
added works page content

### PR DESCRIPTION
**Pull request by myrcutio**

### CODE REVIEW

File: app/src/Routes/Works/Works.js [hunk 1]
```diff
@@ -1,9 +1,26 @@
 import React from 'react';
 import Typography from '@material-ui/core/Typography';
+import * as works from './works.json';
+
+const Piece = (piece) => (
+    <section>
```
Comment (steppy452) [inline app/src/Routes/Works/Works.js:6]: this maybe should be a `list` or something

File: app/src/Routes/Works/Works.js [hunk 2]
```diff
@@ -1,9 +1,26 @@
...
+        <div>{piece.title}</div>
+        <div>{piece.type}</div>
+        {piece.instruments ? piece.instruments.join(",") : "" }
+        <a href={piece.detailLink} target={piece.external ? "_blank" : ""}>Details</a>
+    </section>
+)
+
+const WorksCategory = (categoryName, category) => (
+    <figure className="category">
```
Comment (steppy452) [inline app/src/Routes/Works/Works.js:15]: `figure` doesn't really make sense here. `figure` is typically used for images w/ an optional caption. but i'm also don't think nesting `section` or `list` elements within the `figure` element is valid

File: app/src/Routes/Works/Works.js (hunk 1 above)
Comment (myrcutio) [reply app/src/Routes/Works/Works.js]: makes sense, done

File: app/src/Routes/Works/Works.js (hunk 2 above)
Comment (myrcutio) [reply app/src/Routes/Works/Works.js]: sounds good, switched to an li

File: app/src/Routes/Works/Works.js [hunk 3]
```diff
@@ -1,9 +1,30 @@
 import React from 'react';
 import Typography from '@material-ui/core/Typography';
+import * as works from './works.json';
+
+const Piece = (piece) => (
+    <li key={piece.title + piece.type}>
+        <div>{piece.title}</div>
+        <div>{piece.type}</div>
+        {piece.instruments ? piece.instruments.join(",") : "" }
```
Comment (steppy452) [inline app/src/Routes/Works/Works.js:9]: oh one more thing - i think i'd do something like this instead here:
`{piece.instruments && piece.instruments.join(',')}`

**Pull request merged**